src/depth_sorter.ts

```diff
@@ -23,7 +23,7 @@ function computeDepthShader(itemsPerThread: number, numQuadsUnpadded: number): s
 @group(0) @binding(1) var<storage, read_write> depths: array<f32>;
 @group(0) @binding(2) var<uniform> projMatrix: mat4x4<f32>;
 
-@compute @workgroup_size(1)
+@compute @workgroup_size(64)
```

Comment:
1) I was picking these workgroup sizes semi-arbitrarily. Is there some logic to it? Can we have a comment on that? 

2) If this is whole block is to be gone anyway with radix sort, don't bother with this bullet point. Otherwise, if we're using the constant 64 here and on line 284, can we unify them into a single global const, to avoid accidental mutation of one without the other? I tried to get this functionality with [overridable declarations](https://www.w3.org/TR/WGSL/#pipeline-overridable) but I couldn't get it to work, so I just injected TS consts into shaders via `${CONST_VALUE}`.

Reply:
64 is a recommended value "unless you are targeting a specific video card" as told by webgpu authors, see https://surma.dev/things/webgpu/